app/debrief_gui.py

```diff
@@ -90,106 +108,84 @@ async def download_json(self, request):
             return web.json_response(dict(filename=filename, json_bytes=operations))
         return web.json_response('No operations selected')
 
-    def _build_pdf(self, operations, agents, filename, sections):
+    async def upload_logo(self, request):
+        data = await request.post()
+        logo_file_info = data['header-logo']
+        if logo_file_info:
+            logo_file = logo_file_info.file
+            content = logo_file.read()
+            filename = logo_file_info.filename
+
+            # Write file to uploads directory
```

Comment:
good place for a separate function. Comment highlights it is a separate section

Reply:
separated them out